python/rmmbr/crypto.py

```diff
@@ -0,0 +1,33 @@
+from hashlib import sha256
+
+from cryptography.fernet import Fernet
+from rmmbr.serialization import FunctionSerializer, Serializable
+
+__all__ = ["generate_encryption_key", "EncryptorFunctionSerializer"]
+
+
+def generate_encryption_key() -> str:
+    return Fernet.generate_key().decode()
+
+
+class EncryptorFunctionSerializer(FunctionSerializer):
```

Comment:
i've yet to see a legitimate use case for a class, maybe today wil be the day, but it seems you can just return a single function, avoid the inheritance and it will be shorter and more readable. (instead of using super just reuse functions)

Reply:
👍